src/app.js

```diff
@@ -335,6 +335,10 @@
     onDeviceReady: function() {
       $(document).on("backbutton", spiderOakApp.onBackKeyDown);
       $(document).on("menubutton", spiderOakApp.onMenuKeyDown);
+      // Colored status bar for Lollipop+
+      if ($.os.android && parseFloat(window.device.version) >= 5.0) {
```

Comment:
Need a condition for window.device being undefined, eg in the case of desktop browser. So:

> if ($.os.android && window.device && parseFloat(window.device.version) >= 5.0) {